WordPress/wporg-developer
# Tools: Update node, yarn, composer packages

This updates the node version & yarn packages to use the latest build scripts. `@wordpress/scripts` and `@wordpress/env` are the only packages used here, both have been updated. Since I was updating things, I also updated the composer lockfile.

I tested with my local env and a fresh install, the build works, the env works, and parsing still works (on the fresh install, my existing env is wonky).

**To test**

- Use `nvm install` or `nvm use` to switch to Node 20
- Run `yarn` to update the dependencies
- Run `yarn build` to build the theme
- Start up the env
- Everything should work as expected, you should _not_ have to reconfigure anything
